src/main/java/com/es/phoneshop/model/product/ArrayListProductDao.java

```diff
@@ -1,10 +1,8 @@
 package com.es.phoneshop.model.product;
 
-import java.math.BigDecimal;
-import java.util.ArrayList;
-import java.util.Currency;
-import java.util.List;
-import java.util.NoSuchElementException;
+import org.w3c.dom.ls.LSInput;
```

Comment:
Remove unused imports: Ctrl + Alt + O in IntelliJ IDEA or configure ""optimize imports" option in IDE settings.